components/RatingsTable/index.tsx

```diff
@@ -50,111 +86,134 @@ Props): JSX.Element {
     previousPage,
     gotoPage,
     state: { pageIndex },
-  } = useTable(
-    { columns, data, initialState: { pageIndex: 0, pageSize: 15 } },
-    useSortBy,
-    usePagination
-  );
+  } = table;
+
+  const hasData = rows.length > 0;
 
   return (
     <>
-      <TableContainer>
```

Comment:
looks like a lot of changes.  In reality, I just wrapped the code to depend on if the data is loaded and for pagination.